Jenkinsfile

```diff
@@ -154,7 +154,8 @@ String unit_packages() {
                            'libipmctl-devel python36-pyxattr ' +
                            'python36-tabulate numactl ' +
                            'libyaml-devel ' +
-                           'valgrind-devel patchelf'
+                           'valgrind-devel patchelf ' +
+                           'libunwind'
```

Comment:
```suggestion
                           'valgrind-devel patchelf'
```

Reply:
@brianjmurrell and @ashleypittman , do you mean that in CI there is no more local build of DAOS but only RPM-based installs ?? 
Then I may have to add libunwind detection in utils/sl/components/__init__.py and only configure/build Argobots with libunwind support if present and print a warning message if not ?

Reply:
No.  To clarify, `unit_packages()` is a list of packages to install during unit tests to which you added `libunwind`.  However that list already includes `argobots`: https://github.com/daos-stack/daos/blob/792a88a4c31784146c7e4740f070d7d653d9f874/Jenkinsfile#L147

That `argobots` reference will automatically include `libunwind` without it needing to be named explicitly here.

Reply:
CI will do either local build or RPM based builds depending on if you set Quick-build or not.

Default is local build, and for that you need to ensure the package is in the dockerfile which you've done, if you set Quick-build:true then it'll use rpms and for that the argobots rpm needs to be built correctly, either way it should use the same features so no need to modify components at all.